src/app/commons/modals/RemoveConfirmationModal.js

```diff
@@ -1,25 +1,28 @@
-import './RemoveExamineeConfirmationModal.scss'
+import './RemoveConfirmationModal.scss'
 import React, {createRef} from "react";
-import {renderModal} from "../../commons/modals/modal";
-import {ModalHeader} from "../../commons/modals/ModalHeader";
+import {renderModal} from "./modal";
+import {ModalHeader} from "./ModalHeader";
 
-const RemoveExamineeConfirmationModal = ({title, content, show, onClose}) => {
+const RemoveConfirmationModal = ({title, content, show, onClose}) => {
     const closeIconRef = createRef(), formRef = createRef();
 
     const handleSubmit = e => {
         e.preventDefault();
     };
 
     return renderModal({
-        modalClassName: "remove-examinee-confirmation-modal",
+        modalClassName: "remove-confirmation-modal",
         modalWidth: "480px",
         show, onClose, closeIconRef,
         contentRendering: () => (
             <form onSubmit={handleSubmit} ref={formRef}>
                 <div id="modal" className="font-poppins has-text-centered">
                     <ModalHeader className="header" title={title} textAlign="center"/>
-                    <p>Name: {content.name}</p>
-                    <p>Email: {content.email}</p>
+                    {
+                        content?.map(item =>
+                            <p>{item.title}: {item.value}</p>
```

Comment:
You must add a `key` to item to distinguish them 
read [this](https://reactjs.org/docs/lists-and-keys.html)

`<p key={item.title}>{item.title}: {item.value}</p>`